src/data.js

```diff
@@ -2064,6 +2064,29 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'David Leuliette',
+    description: 'Freelance React Native developer and bootcamp instructor specializing in cross-platform application.',
+    url: 'https://davidl.fr/uses',
```

Comment:
Hi, I tried to open this url but it said, "404 Not Found Sorry, this page does not exist anymore"
Could you please verify this once. Thanks!